### ISSUE TITLE
[review] Dark Mode -- Figure Contrast for Transparent Section of Image

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/quantecon-book-theme#37]
Is there a way to control background for the transparent components of the generated images?

See first generated image in

https://quantecon.github.io/lecture-python-programming.myst/numba.html

with `dark mode` enabled. 

The axis data is a bit difficult to read.

**Comment by DrDrij** [comment]
Yes we can, we can use a filter to invert the colours. Here are two examples from that page:

<img width="973" alt="image" src="https://user-images.githubusercontent.com/5886045/95722885-3e18d100-0cc0-11eb-93fa-cfa2ea3f8d2e.png">


<img width="1076" alt="image" src="https://user-images.githubusercontent.com/5886045/95722942-4d981a00-0cc0-11eb-9c3f-80dd9fe91779.png">

**Comment by mmcky** [comment]
thanks @DrDrij that looks like a big improvement

**Comment by mmcky** [comment]
@DrDrij - just fyi - the preview on that commit doesn't seem to have taken effect for image

https://5f8437440d7ad18f9eee916c--hungry-lovelace-7d0512.netlify.app/matplotlib.html
